Models/Staff.cs

```diff
@@ -1,19 +1,22 @@
+using System;
 using System.ComponentModel.DataAnnotations;
 
 namespace stunning_robot_HR.Models
 {
     public class Staff
     {
         public int Id { get; set; }
-        [Display(Name = "First Name")]
-        public string FirstName { get; set; }
-        [Display(Name = "Last Name")]
-        public string LastName { get; set; }
         
-        [Display(Name = "Total Hours Worked")]
-        public int TotalHoursWorked { get; set; }
+        [Display(Name = "Full Name")]
+        public string FullName { get; set; }
+        
+        [Display(Name = "Date of Birth")]
+        public int DateOfBirth { get; set; }
```

Comment:
I believe this relates to the referenced datetime element in slack. It feels like this should be a DateTime type field rather than an int. What is driving you to utilize int?